content/projects/gsoc/2023/projects/add-probes-to-plugin-health-score.adoc

```diff
@@ -9,36 +9,57 @@ tags:
 - plugins
 status: "Selected"
 sig: platform
-//FIXME: create bio for Jagruti Tiwari in content/_data/authors/
-student: "null_gsoc_contributor"
+student: "jagruti"
 mentors:
 - "alecharp"
 - "dheerajodha"
 - "mostafaashraf"
 links:
     idea: /projects/gsoc/2023/project-ideas/add-probes-to-plugin-health-score
-//   gitter: "some gitter channel"
-//   draft: link to the draft/final proposal
-//   meeting: link to internal section of this page with the meeting details and notes
+   gitter: https://app.element.io/#/room/#jenkinsci_GSoC-Plugin_Health_Score:gitter.im
+   selected proposal: https://docs.google.com/document/d/14URAapu7svkj1pgw3yoCybwn766Net9SZ9coC4zx7yg/edit
+   meeting notes: https://docs.google.com/document/d/1QcwSiAuQtoy4dGlPXgY3w8FjDzTJn-3yCv75U-OFJ04/edit
```

Comment:
The `meeting notes` tag doesn't seem to work. I recommend (while I am researching it) to just use the `meetings` tag and make it point to the `Meetings` section in the document:
```suggestion
   meetings: /projects/gsoc/2023/projects/add-probes-to-plugin-health-score/#office-hours```